### ISSUE TITLE
Fatal Error on Invoice Excel Export

### ISSUE THREAD

**Issue by pranavcp** [husseycoding/customordergrid#11]
Downloaded extension from https://www.magentocommerce.com/magento-connect/custom-order-grid.html

Showing fatal error Fatal error: Call to a member function addData() on boolean in magento-de\app\code\community\HusseyCoding\CustomOrderGrid\Model\Observer.php on line 49
while Invoice Export CSV or Excel XML.

I have added Billing Company column into order grid, that's it and the same is not exporting into CSV.

**Comment by husseycoding** [comment]
Are there any other extensions affecting the order grid?  Have you tested on a clean install?

**Comment by pranavcp** [comment]
Yes, only order delete is installed but I don't think this is causing the
issue. This is from observer and while rendering the layout.

*--*

*Regards,*

*Pranav Patel*
*Sr.Software Engineer,*

**Comment by husseycoding** [comment]
Ok no worries - leave it with me and I'll test/fix it for the next release.

**Comment by pranavcp** [comment]
Ok, can you give me the solution, I am in middle of the project. While I am
removing the line it adds extra column of status and other one in export
CSV.

*--*

*Regards,*

*Pranav Patel*
*Sr.Software Engineer,*

**Comment by husseycoding** [comment]
We will look at it for the next version - I can't say when this will be exactly.  if you need to determine a fix for the issue you are seeing right now then you will need to debug and resolve it yourselfl.  This ticket will be updated once the problem you have brought up is looked at to determine whether if it's an install specific, or extension issue.  If it is indeed an extension issue we will of course fix it.
